libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -688,7 +737,6 @@ void AC_Avoid::adjust_velocity_circle_fence(float kP, float accel_cmss, Vector2f
                 const float max_speed = get_max_speed(kP, accel_cmss, distance_to_target, dt);
                 if (max_speed < desired_speed) {
                     desired_vel_cms *= MAX(max_speed, 0.0f) / desired_speed;
-                    _last_limit_time = AP_HAL::millis();
```

Comment:
Won't this mean `limits_active` will give the wrong answer now?

Do we need a separate `_last_limit_z_time`?

Reply:
@IamPete1 this bit refactors the code that you added sometime back for sailboats. I hope this does not bring any functional changes...

Reply:
Just tested, tacking on fence still works for sailboats.

Reply:
@IamPete1 ah, fantastic, thanks for testing for us.